charactersheet/charactersheet/templates/dm/dmscreen.tmpl.html

```diff
@@ -0,0 +1,333 @@
+<div class="row">
+  <div class="col-lg-6 col-md-6 col-sm-12 col-xs-12">
+    <table class="table table-responsive table-ac-bordered table-hover">
+    <caption>Conditions</caption>
```

Comment:
Can we make this pop out more? The text is subdued, which makes it less scan-able.